tests/data/expression.diff

```diff
@@ -199,7 +213,7 @@
 +    .filter(
 +        models.Customer.account_id == account_id, models.Customer.email == email_address
 +    )
-+    .order_by(models.Customer.id.asc())
++    .order_by(models.Customer.id.asc(),)
```

Comment:
this too

Reply:
Again, because it's a diff-of-diffs it's confusing, but this is expected behavior: the trailing comma is preserved, but because it's in parens it's not exploded (because it looks like a one-tuple).